src/theme/foundations/radii.ts

```diff
@@ -1,4 +1,8 @@
 const radii = {
+  button: {
+    default: "2px",
+    pill: "20px",
```

Comment:
Why not just use the `pill` key below? Or update any existing radii that use `pill` to now be `button.pill` (such as in the `src/theme/components/button.ts` file line 72) and remove the `pill` statement on line 7.

Reply:
I had visions of creating a separate ticket to handle the change, but I can do it here.